iptables/feature_detect.go

```diff
@@ -57,19 +58,21 @@ type Features struct {
 }
 
 type FeatureDetector struct {
-	lock         sync.Mutex
-	featureCache *Features
+	lock            sync.Mutex
+	featureCache    *Features
+	featureOverride *map[string]string
```

Comment:
No need for a `*map` in Go.  maps are a pointer type in go so `map[string]string` can be nil.

Whatsmore, the nil map behaves like an empty map for lookups so you don't need to check if its nil before doing:
```
if value, ok := d.featureOverride["SNATFullyRandom"]; ok {
...
```